frontend/demo/component/richtexteditor/rich-text-editor-set-get-value.ts

```diff
@@ -0,0 +1,55 @@
+import 'Frontend/demo/init'; // hidden-full-source-line
+
+import { html, LitElement, customElement, internalProperty, query } from 'lit-element';
+import '@vaadin/vaadin-rich-text-editor/vaadin-rich-text-editor';
+import '@vaadin/vaadin-text-field/vaadin-text-area';
+
+import { applyTheme } from 'Frontend/generated/theme';
+import { RichTextEditorElement } from '@vaadin/vaadin-rich-text-editor/vaadin-rich-text-editor';
+import { TextAreaElement } from '@vaadin/vaadin-text-field/vaadin-text-area';
+
+@customElement('rich-text-editor-set-get-value')
+export class Example extends LitElement {
+  constructor() {
+    super();
+    // Apply custom theme (only supported if your app uses one)
+    applyTheme(this.shadowRoot);
+  }
+
+  @internalProperty()
+  private htmlValue = '';
+
+  @query('vaadin-rich-text-editor')
+  private richTextEditor?: RichTextEditorElement;
```

Comment:
```suggestion
  private richTextEditor!: RichTextEditorElement;
```
Since you're using `@query()` decorator to assign a value to this property, use the definite assignment assertion operator, `!` instead of marking it as optional with `?`. That way, you won't have to use the optional operator any time you want to reference this variable.

Reply:
Done.